retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/view/ResponseView.java

```diff
@@ -35,12 +42,35 @@ public void setCards(List<Image> images) {
         recyclerView.setAdapter(adapter);
     }
 
+    public void initializeLoader() {
+        getActivity().getSupportLoaderManager().initLoader(0, null, new LoaderManager.LoaderCallbacks<Cursor>() {
```

Comment:
And to avoid the callback, your presenter should be implements LoaderManager.LoaderCallbacks<Cursor>